mobile/ios/EnteMemoryWidget/EnteMemoryWidget.swift

```diff
@@ -90,7 +90,7 @@ struct FileEntry: TimelineEntry {
Fh9rCDJrrq5LkkTTNw3wGzsu0fIiQ3nD2GkAMPdN1AmlRWpMl1IL42x8l3DoAAnLtSXLK28bwCehcOmxc8MHie70X0BhjNi3IQAv7FESmAGpXEG57fFij59l3bCTJ2u4oOFLPAdN1VqwIptcl+A4YyYjLAT8uZGvvoIPS8uZCjUQZip+wbY/bOfvsSsPOeeJ4RdT5G1MfK+gTrZHECafNWyYK/K1EYfdklFD2Em7xtg0+JnzMib/pa9EzMdu06p1AHkfQPL5IMb6Ro79PKCpW/F0vKL3+KZTc+rn7/HkFB0grAX2ZvRiXKqGn0psERlqHfo035L5iqV7lRlZpFv6LH6voHoWDbITmnf4OfoXCJMiFFq85c9uAVnscooUClYg9gkmD9YtLh4bwG7IlqjgzALLn6LC2OGfA2dsh364+pPqQpj5CypgjBa2GfqFBdsSJ0skGiTxaSz8hGzQCbvGxDOgoAzpU0uJwwWMD+5CCBPV+lRlKwiVBFLO5GJFBf56UJIyaCPMGtu4I6hjJ5mtNkBRa82FYE0YjkKeorluhElLlIHYUiRjwrISQqRERMFGX/+VIOrCQN7+NRbOygjVqWYx04TFTG+1S4LKS3DVEX6CCuz5gbucVFcXtzt4nY95Ev984na1NeoXFPOh/1fWcGiV5Ozik/qmQTuI436Io1YmBWEjYuezP7IN7q2NwX58qvviEMdc4kgrIheyhS4yu8rytQWUuQjwOw2YkpPpouxjSHnS8MvizJi2crRw0yDIWWOMPMXv4nFR5hexkLXm9p1m1a+kPcNZFOKtm4BcZZGMwapaj2E5VAW3MAtz4sN7mt1/O4jzXwhjVg266Py09PTi4oWL84sPpGR8dsC83wRJ31tMmISE+OndLU8QnQSzg9wx8YEbckfZcR6CgvLyjBmeDA9va8oa/HiYkBigV4SUyk9fJm9bwBno3Pnusk573oRPtrJGnfgE05Xqau+uaKiJ6eqsbEme1FhYVfZGXTrSQjEYhCIpBJH6SPMyDGXWhgL7HaPEXuI2cOmfWk9TNeSR3V1RUVFDiAxO7tQayAK2ZQ0N2rVLKlKpyABvVndISPGqA0qTMewkV/Xsj0yEKtBIOZoCMRBvd/KesIYmaVRe9it63f9lNlvOF6PdyFf/GYQ1SCbogKxy+GUKOoH5653WNdzdMnvk1XBZPIN3JaBCcS6JL0qMWac9WD7waqUrQgFXXqnL0n3DTgNo9cNaF3ZVKXssVDQpakjpemYS8dRaF1rCtcPzl0+N+S6ftdDB2QFXo93UbmB20lUUzZoDmULv3b93FDq+odeDCglmY4GHPniN0eRrTbgYBVl1CiGs42WxxaKLZ26Q+coU2gDzD/OVsSPF5ajd3fbLBO/fO5AADujFEojZhdf19ocYEyCSRCGPsCpVPbIxF0PPfdNgDj+pyPEIii9/YeQ4KYu+wOMneqe7BcflrN6/Z1WykSaq9s+otShXSiGyojpjK8uzQsOFsAve3rDow3r4bk5JlTHri+vp6kKUBqhfVXClUasZx5l5eKlBuGFwnDtbT997oeHvtylnaXfffnQ9c/99LZvTmlmioX2jo4LjVgFvEY6LCpEfMFtcgMlpMcZOPXRgQPf3Hbbiz8FuJ7Hc/D/b6Nx4MBHp05RRpDXWZ+nZ3jFRz5ULdSpsCInex5j/0MVX0ySGUuu74B3W4QYwY625OSgng6BK4zYTdU9OTXZXaLVmvHRIeMLlvGlyQBLe0NKWrC3DX7bUl0tHSeN2E035FTVLOqaJ/9NxvvGjXOCMIC1Y0OhYC2PY4slzKdHIoXciFXTRGUvKpuH/FX8CSGlixkdG0uWYO3S3g3aaQtLjNPH1sDQ0jbRd9OVEtF7xOwiqienMbuwDPNr+GMjY6ISw20XhdG+qKi4mJiYSZEAk2LALPdQMgJtS8c6h4IDqsSNxwpNVGQNjbXJvk+erq2LCeSzdy0qUVXSEiX5fNJExUUl2l60ouPjJsVO8GOeYzIJ9UvHOjqHSoPBgRIUezHMbT29Y50bBghUDZT2jtUj3r1DX5fbhztc25rMhypRLFETIxPi4u2PKHqSwxcXOSGM/MnvTdaMtvr6+qUsOuCIWkQs+BwEubDs7SylyQ0GS0pK6H+Wlg51dozVt2Hfr0RfEy6c/k3m1VlNVCO2RFH+iXTmi48OD02BCvdNitCUqtYmG0CbxL51GnmL5CG9TbgEq5bEbiKWqDBYonzRoZQT4fGxYZqLiwHG6mUZ0ABjbUP6OlNsTpxXbUqcq5SohFCUKETRignTpd10P+5HFe2RgXq9b8FRHkbn1ciYuCifTz33gBDrqrO6RPlDVqIw0RWp2xuVdOh61kHUe5TqoezRUtyHPCKClq84+uDJq9m6+kekEjU+lCUK3yAMM2JnS0of1ZjJeoO49ygdM/0WcvpA9AH+4n2J8MHCWxW6qjWVqC4XlSiVhq4xlGzoUKtmj6p0sPI29KrwXt8xNGC8S+JnJO+1OYZK1ESnSpSuJS6dGNgw1Dv26Np6uZ5f29FZOpCn8S06O2TvQH/9ox29Q1rfQR2FOcoSVUYsUY5nPoKQsga0lxoIAsdcUmK0a1gCYU9/8tpFVT0VPbBEEbNqZEJMTFQUnVNpJEbTcBlh4ZQHTdURYEIEi0gxJsVIEcXBxwLQbl3AJnhshBA87QzRcTCUE/WFsceYS2J4guBQtM6JenANfRMiJsVhiYuO8ShzaQbFbSaI9qLMlXxFxmsdE/XgaDkbHwksu6qw9CVMPB/19vjxsZHAPQHNJQHQ2JGRsePHT/T7w1waQzQmcr8A/fPrVYz0YvNk15PmnxhL/3rwl9NnbsKBH4LOKF5gFCBBMFU0vUr48UC9XGzSuG8RJ/Zm0JBb58jD42PGh7mRJSB53doxchyJrmCNaZTHu64B61bQIx4J4/0OhZMXTYZpo6MtMgS8ceLICyersqQvLibWcuKghPCKEw7/H8HYQ8U89FbDAAAAAElFTkSuQmCC"
```

Comment:
There should be away to include these assets separately instead of using base64. Is that not possible due to any particular limitation.?

Reply:
Done!